### ISSUE TITLE
`selectInput` renders "empty" choice with a very tiny row

### ISSUE THREAD

**Issue by aronatkins** [rstudio/shiny#900]
Given the control:
```
selectInput("a","b",c(`use default="", "east", "west", "north", "south"))
```

you get an input selector with a very short row representing the default choice. See screenshot.

<img width="307" alt="screen shot 2015-07-16 at 1 03 18 pm" src="https://cloud.githubusercontent.com/assets/362187/8731933/6ad17cc6-2bc8-11e5-8546-6c92e138da03.png">

**Comment by yihui** [comment]
Hmm. I don't see that tiny row on Ubuntu/Chrome. Your OS/browser?

```r
library(shiny)
shinyApp(
  ui = fluidPage(
    selectInput("a","b",c(`use default`="", "east", "west", "north", "south"))
  ),
  server = function(input, output) {}
)
```

![screenshot- -dropbox-repo-knitr - shiny](https://cloud.githubusercontent.com/assets/163582/8735896/e66cc564-2bd9-11e5-9943-48accdb279da.png)

**Comment by aronatkins** [comment]
I'm sorry. I gave you the wrong sample code. This happens only when I try and use a single-space string as input. I wrongly recalled my state when I grabbed the screenshot.

Here's the actual snippet that causes the bad rendering:

```
selectInput("a",NULL,c(" ", "east", "west", "north", "south"))
```

Making the first element an empty string and/or naming that element avoids the problem.

Maybe this is enough of a fringe issue that this can be closed.
